src/Driver/OCI8/Result.php

```diff
@@ -73,7 +73,7 @@ public function fetchFirstColumn(): array
         return $this->fetchAll(OCI_NUM, OCI_FETCHSTATEMENT_BY_COLUMN)[0];
     }
 
-    public function rowCount(): int
+    public function rowCount(): int|string
```

Comment:
What's the reason for this change? According to the docs, `oci_num_rows()` returns `int|false`, so we should not encounter a string here. If I recall correctly, the only driver returning row counts as string under certain conditions is MySQLi.

Reply:
I changed it consistently with the changes in the interface. But yeah, let me revert the unnecessary changes.